posts/start-web-performance-first-steps.md

```diff
@@ -0,0 +1,120 @@
+
+Since we started measuring and tagging our users based on the performance, we've been able to better understand how their behavior are impacted by the performance they had when seeing and using a feature.
+
+[We've written about this](/posts/avoid-losing-users-and-conversions.md) and also [there's a lot of relevant content about it](#see-also), so I'll just leave the main points here:
+
+1. Start collecting [Real User Metrics](https://web.dev/user-centric-performance-metrics/) as soon as possible;
+2. Track this in a way you can correlate it with your business and user metrics;
+3. Don't use only average and median a lot of your users are better described by the higher percentiles (p75, p90, p99, p99.99);
+4. Build discoverable, accessible and easy to use dashboards showing this metrics and how they relate to your business.
+
+### Lock: Audits and Continous Improvement
+
+Locking here means trying our hardest to catch code that can degrade performance before it goes into production, hence working with alarms, audits and mainly [Constinous Improvement (CI)](https://en.wikipedia.org/wiki/Continual_improvement_process).
+
+When we think about this, we usually understand the degrading side and why we'd like to block that, but we oftentimes miss the main point that **better audits and CI are our greatest tools for automatic knowledge sharing**. When we design our audits and CI, we must understand that and work towards sharing knowledge and raising the bar of the company as a whole. When engineers feel that the constraints and limitations are quality standards that will enable them to grow and not hinderances, we've met our goal with our CI pipeline.
+
+> **Better audits and CI are our greatest tools for automatic knowledge sharing**
+
+We're still dillengently working in our audits and CI pipeline and far from perfect, but our performance tests currently consists of:
+
+1. [Lighthouse CI Performance Budgets](https://web.dev/use-lighthouse-for-performance-budgets/): We use these to prevent increases in Javascript size, Image Size, Requests Counts and Total Size of each of our main entry points -- Our [Homepage](http://quintoandar.com.br/), our [Search Page](http://quintoandar.com.br/alugar/imovel/sao-paulo-sp-brasil), and our [Listing Page](https://www.quintoandar.com.br/imovel/892879684). This is our most important audit and the one that gives us more safety, but the others allows us to better understand the issues;
+2. [Danger](https://github.com/danger/danger) script for file sizes: While the most important think is measuring the impact in each route separately, distinct file sizes also impact our users in different ways, so we track the sizes on our must important files, either because they're very common on different pages or why they're already big, and try not to increase them too often;
```

Comment:
How about some screenshot of our CI's console w/ a broken performance step

Reply:
While the most important **think** (thing)

Could not find it on the files changed